hub/dune

```
@@ -0,0 +1,6 @@
+(executables
+ (libraries cohttp-lwt-unix github_unix cmdliner yaml yaml.unix)
```

```suggestion
 (libraries cohttp-lwt-unix github-unix cmdliner yaml yaml.unix)
```

to match the public name of the library